CONTRIBUTING.md

```diff
@@ -105,18 +114,18 @@ tools:
   $ pytest --cov=. path/to/tests_for_package
   ```
 
--  No pyflakes warnings, check with:
+-  No style warnings, check with:
 
   ```bash
-  $ pip install pyflakes
-  $ pyflakes path/to/module.py
+  $ pip install flake8
```

Comment:
Is there a reason to not use the bash script `ci_scripts/flake8_diff.sh`?

Reply:
Discussed in person: This will not generally work in Windows environments. Added a note in the script to update documentation if things change.